contracts/nftauction.js

```diff
@@ -222,6 +288,7 @@ actions.create = async (payload) => {
     minBid,
     finalPrice,
     priceSymbol,
+    fee,
```

Comment:
feePercent, variable should indicate some form of units. Documentation will clarify that it is percentage*100 , max 10000

Ditto for market parameter